libs/clients/cms/schema.graphql

```diff
@@ -1,7 +1,10 @@
+directive @contentSourceMaps on QUERY
```

Comment:
Why did this file get updated, are you using the Contentful graphql api to fetch data?

Reply:
This was probably me throwing the kitchen sink at the wall, trying to get anything to run. 

Reverted